Gopkg.toml

```diff
@@ -89,3 +93,8 @@ ignored = [
   branch = "master"
   name = "sigs.k8s.io/cluster-api-provider-openstack"
   source = "https://github.com/openshift/cluster-api-provider-openstack.git"
+
+[[constraint]]
```

Comment:
@enxebre 
there is fork in openshift org for this?

Reply:
There is, but we are waiting for all PR to be merged. Please accelerate reviews on them for this to be inlined faster. https://github.com/openshift/cluster-api-provider-azure/pulls

@awesomenix 's branch is functional for now and allows testing until these PRs are merged

Reply:
resolving as awaiting for the openshift fork to be ready